app/main/views.py

```diff
@@ -87,25 +85,26 @@ def update_service(service_id):
     db.session.add(service)
     db.session.commit()
 
-    return jsonify(services=jsonify_service(service)), 200
+    return jsonify(services=jsonify_service(service)), http_status
```

Comment:
I think that works with a `Location` header for `POST` where you need to direct the client to a different URL but I don't agree that that either is ok for `PUT`. The resource we've set isn't located at a different URL, it's at the URL we've just interacted with.
I found [this link](https://www.subbu.org/blog/2008/10/location-vs-content-location) useful.